packages/plugins/plugin-markdown/src/capabilities/react-surface.tsx

```diff
@@ -74,7 +74,8 @@ export default () =>
     createSurface({
       id: `${MARKDOWN_PLUGIN}/editor`,
       role: ['article', 'section'],
-      filter: (data): data is { subject: { id: string; text: string } } => isEditorModel(data.subject),
```

Comment:
this should not be removed

Reply:
sorry what do you mean -- I replaced it with the Effect type check -- is that not ok?

Reply:
they're not the same thing, this is handling the case where it is *not* a document